### ISSUE TITLE
Checkbox text should be in label tag

### ISSUE THREAD

**Issue by timworx** [patosai/tree-multiselect.js#25]
Just implemented tree-multiselect.js. It's a really nifty plugin, however, it would greatly help usability if the text for each checkbox was wrapped in an associated label tag. Then you could click the label text or the checkbox to select the checkbox, instead of just the small textbox target area.

For example, in a section titled "Brooklyn"

```html
<div class="item" data-value="Flatbush" style="display: block;">
    <input type="checkbox">Flatbush
</div>
```

With some sort of name-spaced generated ID becomes:

```html
<div class="item" data-value="Flatbush" style="display: block;">
    <input type="checkbox" id="tms-brooklyn-flatbush"><label for="tms-brooklyn-flatbush
</div>
```

What do you think?

**Comment by patosai** [comment]
This seems like it would help. What happens for section checkboxes?

**Comment by timworx** [comment]
I would leave them as they are. A select all for a section should be a very explicit action. You don't want to knock out your current choices accidentally.

 It's a little ambiguous as to whether a click on that label should expand the section or select all, but you can't wrong with the current functionality of extending the section.

**Comment by patosai** [comment]
Implemented in 21ef9490aeb3e67ab2a9cc3bf41680194b2a73e2
